packages/mobile/src/analytics/constants.ts

```diff
@@ -43,6 +43,7 @@ export enum CustomEventNames {
   request_select_recipient = 'request_select_recipient', // when recipient was selected
   request_amount_back = 'request_amount_back', // back button pressed on amount screen
   request_continue = 'request_continue', // when next button pressed on amount enter page
+  request_unavailable = 'request_unavailable', // when request attempted to unverified user
```

Comment:
Please add it as well here https://github.com/celo-org/celo-monorepo/blob/e751e5a851563a14961089cc2e870e320ef9a307/packages/mobile/src/analytics/constants.ts#L237. Otherwise it would be filtered.